Android init (#537)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/sdks/android-kotlin/index.md

### Step 3: Initialization

> Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. We recommend doing the initialization in the Main Activity, which will never get destroyed. Or the Application class if you have one. Then You can use the Android SDK anywhere after it's initialized in an Android application.

Reviewer: Minor adjustments on the sentence.
```suggestion
Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. We recommend doing the initialization in the main activity, which will never get destroyed, or in the `Application` class if you have one. After it's initialized, you can use the Android SDK anywhere in your Android application.
```

Author: The default android project template usually has `MainActivity` class and `MainApplication`. 
Maybe we can use `MainActivity` or `Application`?

> Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. We recommend doing the initialization in the Main Activity, which will never get destroyed. Or the Application class if you have one. Then You can use the Android SDK anywhere after it's initialized in an Android application.

Reviewer 2: Just some small refinements to Marvin's suggestion: 

```suggestion
Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. Amplitude recommends doing the initialization in the Main Activity, which never gets destroyed, or the Application class if you have one. After it's initialized, you can use the Android SDK anywhere in your Android application.
```
